MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Tomography/ITomographyROIPresenter.h

```diff
@@ -58,7 +58,6 @@ class ITomographyROIPresenter {
     ColorRangeUpdated,     ///< The uses changes the values or options of the
                            ///< color range
     SelectCoR,             ///< Start picking of the center of rotation
-    FindCoR,               ///< Run calculate CoR
```

Comment:
These changes were from #18258. I've removed them in that PR as well as they were unused code. The only changes this PR covers is the merge of the tabs